nautobot/extras/api/customfields.py

```diff
@@ -88,7 +88,7 @@ def to_internal_value(self, data):
         return data
 
 
-# TODO: should be CustomFieldModelSerializerMixin
+# 2.0 TODO: should be CustomFieldModelSerializerMixin
```

Comment:
This and the one below also should be implemented as rename/deprecation in 1.5 so that we can remove the old name in 2.2.